templates/node-ts/src/utils/errors/ResponseError.ts

```diff
@@ -0,0 +1,10 @@
+export class ResponseError extends Error {
```

Comment:
Minor but maybe this would be better named like the following since this is specific to the HTTP protocol. What do you think. 
```suggestion
export class HttpError extends Error {
```

Reply:
I like it